locale/fi/info.json

```diff
@@ -89,7 +89,7 @@
         {
           "image.key": "",
           "image.alt": "",
-          "text": "Suomi.fi Design Systemiä koskeva keskustelu käydään ensisijaisesti Suomidigin Slack-yhteisössä (kanava #designsystem). Kehittäjien, komponenttikirjaston käyttäjien välinen keskustelu löytyy Githubista.",
+          "text": "Kehittäjien ja komponenttikirjaston käyttäjien välinen keskustelu löytyy Githubista.",
```

Comment:
While changing this, could fix to use the `GitHub` so the H is capitalised.

Reply:
Thanks. This is now fixed.

Reply:
Issue still exists, the fix was made in different file: `locale/fi/common.json `
But good to have the fix also there though 👍

Reply:
Found a few more instances of the same capitalization issue. Should now be fixed.